### PR TITLE
Rename structured to tabular

### PR DESCRIPTION
**Pull request by elboy3**
https://app.shortcut.com/galileo/story/5026/rename-structured-tabular

### CODE REVIEW

**Comment by Ben-Epstein** [comment]
@elboy3 something seems wrong here. 5000+ files doesn't look right

**Comment by setu4993** [comment]
@elboy3 : You might need to revert your original commit ([5368e3e](https://github.com/rungalileo/dataquality/pull/571/commits/5368e3e413a03dd4ccd737ffe01bf19d13a66ced)) for the updated `.gitignore` to take effect.

File: dataquality/loggers/model_logger/tabular_classification.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from dataquality.loggers.model_logger.base_model_logger import BaseGalileoModelLogger
+
+
+class TabularClassificationModelLogger(BaseGalileoModelLogger):
+    # NOTE: Tabular Classification doesn't require a model logger
+    __logger_name__ = "tabular_classification"
+    logger_config = tabular_classification_logger_config
+
+    def _get_data_dict(self) -> Dict:
+        return {}
```
Comment (setu4993) [inline dataquality/loggers/model_logger/tabular_classification.py:15]: Ha, one of those times when `git mv` isn't smart 😅.

**Review by setu4993 — APPROVED** [review]
Nice refactor! Thanks, Ben!

**Pull request merged**